deploy/kubernetes/kms-issuer.yaml

```diff
@@ -1,515 +0,0 @@
-apiVersion: v1
-kind: Namespace
```

Comment:
This is out of date with the customise. We should also add our helm chart to this repo too. So happy to delete this. Some people might be using it however. 😢

Reply:
Yeah, I think we should keep this but we need a new PR for an automated process when we release something new